components/ai_chat/resources/page/api/index.ts

```diff
@@ -121,7 +121,7 @@ class API {
   }
 
   private dispatchDebouncedStateChange = debounce(() => {
-    console.debug('dispatching uistatechange event', {...this.state})
+    console.debug('dispatching uistatechange event', { ...this.state })
```

Comment:
nit: if you have to push again please can you remove this change as I think it'll cause a merge conflict with my PR